lib/src/routing/connection_registry.rs

```diff
@@ -419,49 +290,26 @@ mod tests {
                 .chain(routers.clone())
                 .collect(),
         };
-        let config = Config {
-            uri: "neo4j://localhost:7687".to_string(),
-            user: "user".to_string(),
-            password: "password".to_string(),
-            max_connections: 10,
-            db: None,
-            fetch_size: 200,
-            tls_config: ConnectionTLSConfig::None,
-        };
-        let registry = Arc::new(ConnectionRegistry::default());
-        let ttl = refresh_all_routing_tables(
-            config.clone(),
-            registry.clone(),
-            Arc::new(TestRoutingTableProvider::new(&[cluster_routing_table])),
-            &[],
-        )
-        .await
-        .unwrap();
-        assert_eq!(ttl, 300);
-        assert_eq!(
-            registry.default_db_name.read().unwrap().clone().unwrap(),
-            "neo4j"
-        );
-        assert!(registry.databases.contains_key("neo4j"));
+        let config = make_config();
+        let registry = Arc::new(ConnectionRegistry::new(
+            &config,
+            Arc::new(TestRoutingTableProvider::new(&[
+                cluster_routing_table.clone()
+            ])),
```

Comment:
There also from_ref for getting a slice from a single element ref

```suggestion
            Arc::new(TestRoutingTableProvider::new(std::slice::from_ref(
                &cluster_routing_table,
            ))),
```